src/Components/Blazor/Build/src/targets/Publish.targets

```diff
@@ -26,9 +26,8 @@
       </ContentWithTargetPath>
 
       <!-- Publish all the 'dist' files -->
-      <_BlazorGCTPDIDistFiles Include="@(BlazorItemOutput->'%(TargetOutputPath)')" />
-      <_BlazorGCTPDI Include="@(_BlazorGCTPDIDistFiles)">
-        <TargetPath>$(BlazorPublishDistDir)$([MSBuild]::MakeRelative('$(TargetDir)dist\', %(Identity)))</TargetPath>
+      <_BlazorGCTPDI Include="%(BlazorOutputWithTargetPath.Identity)">
```

Comment:
`_BlazorGCTPDI` is an extremely obscure name. Could this be renamed to something meaningful?

Reply:
I believe he is following an obscure convention to improve msbuild performance, its Get Copy To Publish Directory Item which is the item without the associated metadata.

Reply:
Yup, as @javiercn pointed out, these are named so in the .NET SDK and we're following their poorly thought out naming convention.